README.md

```diff
@@ -24,3 +24,11 @@ Thereafter, run `script/beer_run.rb` to update things.
 ## TODO
 
 - replace TeXShop with standalone TeXLive
+
+## QMK
+
+- run `qmk setup benlaverriere/qmk_firmware` to get things ready
```

Comment:
should point to the zsa firmware — clarify what else `qmk setup` does, and whether we can just stow the `ini` file instead

Reply:
https://github.com/qmk/qmk_cli/blob/master/qmk_cli/subcommands/setup.py

Reply:
It's tempting to use `qmk setup` directly, in case it does additional things in the future. But since it needs an explicit `--branch` specification in order to work with the ZSA fork of QMK, a submodule seems simpler. I'll add a `qmk doctor` to `beer_run` to keep it up to date.